### PR TITLE
correct grammatical error on animation-name and Element.animate() CSS reference pages

### PR DESCRIPTION
**Pull request by idorenyinudoh**
> What was wrong/why is this fix needed? (quick summary only)

There were grammatical errors that could lead to confusion

> Issue number (if there is an associated issue)

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/web/css/animation-name/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 ---
 <div>{{CSSRef}}</div>
 
-<p>The <strong><code>animation-name</code></strong> <a href="/en-US/docs/Web/CSS">CSS</a> property specifies the names of one or more {{cssxref("@keyframes")}} at-rules describing the animation or animations to apply to the element.</p>
+<p>The <strong><code>animation-name</code></strong> <a href="/en-US/docs/Web/CSS">CSS</a> property specifies the names of one or more {{cssxref("@keyframes")}} at-rule or at-rules describing the animation or animations to apply to the element.</p>
```
Comment (wbamberg) [inline files/en-us/web/css/animation-name/index.html:14]: I think "one or more at-rules" is fine here.

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for your PR, @idorenyinudoh ! I like the first change but think the second one (to animation-name) is not needed.

**Comment by idorenyinudoh** [comment]
> Thanks for your PR, @idorenyinudoh ! I like the first change but think the second one (to animation-name) is not needed.

Yeah that's one a mistake. My bad.

File: files/en-us/web/css/animation-name/index.html (hunk 1 above)
Comment (wbamberg) [inline files/en-us/web/css/animation-name/index.html:14]:
```suggestion
<p>The <strong><code>animation-name</code></strong> <a href="/en-US/docs/Web/CSS">CSS</a> property specifies the names of one or more {{cssxref("@keyframes")}} at-rules describing the animation or animations to apply to the element.</p>
```

**Review by wbamberg — APPROVED** [review]
👍 thanks for the fix @idorenyinudoh !

**Pull request merged**